src/app/globals.css

```diff
@@ -30,6 +31,7 @@ body {
 body {
   color: --foreground-color;
   background: --background-color;
```

Comment:
change value to `var(--foreground-color)`
change value to `var(--background-color)`